generators/app/templates/src/controller/Main.controller.ts

```diff
@@ -0,0 +1,14 @@
+import MessageBox from "sap/m/MessageBox";
+import Controller from "sap/ui/core/mvc/Controller";
+import AppComponent from "../Component";
+
+/**
+ * @namespace <%= appId %>.controller
+ */
+export default class MainController extends Controller {
```

Comment:
In the meantime, the hello world app was changed to have the controllers just named "App", "Main" etc.:
`const App = Controller.extend("ui5.typescript.helloworld.controller.App", {`

Otherwise it's a bit strane when the class is named ...controller.AppController, but the UI5 name of it as used in the XMLView is named ...controller.App.

But then again, this is only a local name and not a named export, so we could go either way. Not sure...

This applies to both controllers.